policy/modules/kernel/files.te

```diff
@@ -75,6 +75,19 @@ optional_policy(`
 type etc_runtime_t, configfile;
 files_type(etc_runtime_t)
 
+#
+# fstab_t is the type of /etc/fstab
+#
+type fstab_t, configfile;
```

Comment:
I'm still not on board with this. I'd prefer to split this out to a separate PR.

Reply:
OK. I'll split it out into its own PR and also open an issue where I'll re-address my concerns.